common/presenters/move-to-card-component.js

```diff
@@ -2,21 +2,32 @@ const i18n = require('../../config/i18n')
 
 const personToCardComponent = require('./person-to-card-component')
 
-function moveToCardComponent({ showMeta = true, showTags = true } = {}) {
+function moveToCardComponent({
+  isCompact = false,
+  showImage = true,
+  showMeta = true,
+  showTags = true,
+} = {}) {
   return function item({ id, reference, person = {}, status }) {
     const href = `/move/${id}`
     const excludedBadgeStatuses = ['cancelled']
-    const statusBadge = excludedBadgeStatuses.includes(status)
-      ? undefined
-      : { text: i18n.t(`statuses::${status}`) }
-    const personCardComponent = personToCardComponent({ showMeta, showTags })({
+    const statusBadge =
+      excludedBadgeStatuses.includes(status) || isCompact
+        ? undefined
+        : { text: i18n.t(`statuses::${status}`) }
+    const personCardComponent = personToCardComponent({
```

Comment:
Is there a reason for the personToCardComponent to not handle the  `isCompact`ness? Just asking...

Reply:
At this stage it was only needed for displaying the full "card" so didn't want to complicate the person presenter just yet.

The person presenter is shared between being used in the card and the person search results. In its current singular use, the person doesn't need a "compact" variation.